docs/guides/improve-tandem.md

```diff
@@ -179,7 +179,7 @@ That said not every task needs to be generated from a discussion. _Generally_ di
 1. Verify a problem
 2. Validate an idea
 3. Expand or hone thinking on a particular task under consideration
-4. Clarify an extand policy or procedure that is confusing
+4. Clarify an extend policy or procedure that is confusing
```

Comment:
```suggestion
4. Clarify or extend an existing policy or procedure that is confusing
```

Reply:
resolved via @pirog in slack; should be `extant` updated PR